certification/internal/engine/engine.go

```diff
@@ -342,22 +357,23 @@ func writeCertImage(img v1.Image) error {
 		DockerImageDigest: digest.String(),
 		DockerImageID:     manifest.Config.Digest.String(),
 		ImageID:           digest.String(),
+		Architecture:      config.Architecture,
 		ParsedData: &pyxis.ParsedData{
-			Architecture:  config.Architecture,
-			Command:       strings.Join(config.Config.Cmd, " "),
-			Created:       config.Created.String(),
-			DockerVersion: config.DockerVersion,
-			ImageID:       digest.String(),
-			Labels:        labels,
-			OS:            config.OS,
-			Size:          size,
-			// TODO: figure out how to get/convert this data from crane
-			UncompressedLayerSizes: layerSize,
+			Architecture:           config.Architecture,
+			Command:                strings.Join(config.Config.Cmd, " "),
+			Created:                config.Created.String(),
+			DockerVersion:          config.DockerVersion,
+			ImageID:                digest.String(),
+			Labels:                 labels,
+			OS:                     config.OS,
+			Size:                   size,
+			UncompressedLayerSizes: layerSizes,
 		},
 		RawConfig:         string(rawConfig),
 		SumLayerSizeBytes: sumLayersSizeBytes,
-		// TODO: figure out how to get/convert this data from crane
-		UncompressedTopLayerId: "placeholder",
+		// This is an assumption that the DiffIDs are in order from base up.
+		// Need more evisdence that this is always the case.
+		UncompressedTopLayerId: config.RootFS.DiffIDs[0].String(),
```

Comment:
Non Blocking: If this is `top` level would this be the _last_ element in the array ie `config.RootFS.DiffIDs[len(config.RootFS.DiffIDs)-1]`?

Reply:
I thought so, but in my findings it's the first in the list.

Reply:
Okay, that's kind of confusing for sure. Happy to test more on this when the time comes.